Hunting Queries/Corelight/CorelightAbnormalEmailSubject.yaml

```diff
@@ -0,0 +1,23 @@
+id: e9441d57-39f4-41fb-aaad-d02e47783d1a
+name: Corelight - Abnormal Email Subject
+description: |
+  'Query searches for emails with NON-Ascii characters within the Subject .'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated